src/overlays/actors/ovl_En_Rz/z_en_rz.c

```diff
@@ -27,27 +27,20 @@ void func_80BFC674(EnRz* this, PlayState* play);
 void func_80BFC7E0(EnRz* this, PlayState* play);
 void func_80BFC8F8(EnRz* this, PlayState* play);
 
-typedef enum {
-    /* 0 */ EN_RZ_ROSA_SISTERS_JUDO,
-    /* 1 */ EN_RZ_ROSA_SISTERS_MARILLA
-} EnRzRosaSisters;
-
-typedef enum {
-    /* 1 */ EN_RZ_TYPE_1 = 1,
-    /* 2 */ EN_RZ_TYPE_2
-} EnRzType;
+#define EN_RZ_STATE_1 (1 << 0)
+#define EN_RZ_STATE_2 (1 << 1)
 
 typedef enum {
-    /* 0 */ EN_RZ_ANIM_0,
-    /* 1 */ EN_RZ_ANIM_1,
-    /* 2 */ EN_RZ_ANIM_2,
-    /* 3 */ EN_RZ_ANIM_3,
-    /* 4 */ EN_RZ_ANIM_4,
-    /* 5 */ EN_RZ_ANIM_5,
-    /* 6 */ EN_RZ_ANIM_6,
-    /* 7 */ EN_RZ_ANIM_7,
-    /* 8 */ EN_RZ_ANIM_8,
-    /* 9 */ EN_RZ_ANIM_9
+    /* 0 */ EN_RZ_ANIM_MIN,
+    /* 1 */ EN_RZ_STANDING_ANIM,
+    /* 2 */ EN_RZ_WALKING_ANIM,
+    /* 3 */ EN_RZ_SITTING_ANIM,
+    /* 4 */ EN_RZ_APPLAUDING_ANIM,
+    /* 5 */ EN_RZ_ON_KNEES_ANIM,
+    /* 6 */ EN_RZ_DANCE_ANIM,
+    /* 7 */ EN_RZ_LINK_NORMAL_WAIT_FREE_ANIM,
+    /* 8 */ EN_RZ_LINK_DANCE_ANIM,
+    /* 9 */ EN_RZ_ANIM_MAX
```

Comment:
Isn't 0 `THINKING`? Also on enums the prefix should be the same for all of them, i.e. `EN_RZ_ANIM_STANDING` rather than as a suffix as it is in the object.